### ISSUE TITLE
error: vrite-app      | No email service configured

### ISSUE THREAD

**Issue by ovizii** [vriteio/vrite#39]
I found this project on reddit: https://www.reddit.com/r/selfhosted/comments/16hm2z0/vrite_v020_opensource_collaborative_developer/ and have been following the discussion over there to get started. 

I can't seem to get the SMTP service setup. All my docker contaienrs are sending through a postfix running locally which serves as a poroxy so no authentication is needed.

Here is my config:

```
SMTP_HOST=10.10.10.10
SMTP_PORT=25
#SMTP_USERNAME=
#SMTP_PASSWORD=
SMTP_SECURE=false
```

However the momnent I try to first register an account I see this error: `vrite-app      | No email service configured`

**Comment by areknawo** [comment]
Currently, the `SMTP_USERNAME` and `SMTP_PASSWORD` are required to enable the SMTP service.

On a side note, I'm still working on the updates and fixes recommended in the post, including better logging to help with issues like this.

**Comment by areknawo** [comment]
You can omit them entirely now (v0.2.1). Just make sure you're running the latest version.

**Comment by ovizii** [comment]
thanks, I'll try again tomorrow as "docker compose pull" does not seem to pull any new image right now.

**Comment by areknawo** [comment]
Sorry for that. It appears there was an issue with the GitHub Action for image publishing.
The new image should be released now, accessible via either the `0.2.1` or `latest` tags.
You can monitor this [here](https://github.com/vriteio/vrite/pkgs/container/vrite%2Fapp).

**Comment by ovizii** [comment]
I just did another pull, got the latest image versions and am now seeing this error when trying to register:

```
vrite-app       | [Error: 589BAF2F697F0000:error:0A00010B:SSL routines:ssl3_get_record:wrong version number:../deps/openssl/openssl/ssl/record/ssl3_record.c:354:
vrite-app       | ] {
vrite-app       |   library: 'SSL routines',
vrite-app       |   reason: 'wrong version number',
vrite-app       |   code: 'ESOCKET',
vrite-app       |   command: 'CONN'
vrite-app       | }
```
these are my SMTP settings:

```
SMTP_HOST=10.10.10.10
SMTP_PORT=25
#SMTP_USERNAME=
#SMTP_PASSWORD=
SMTP_SECURE=false
```

**Comment by areknawo** [comment]
Ok, that's some progress. 🙂
I think there was an issue with passing the `SMTP_SECURE` flag, now fixed in v0.2.2. This should resolve the SSL connection issue. Try to pull the latest image again.

**Comment by ovizii** [comment]
Thanks, problem solved, SMTP works now.
